src/BlockNumbering.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file BlockNumbering.cpp
+ *
+ * @author  Guy Maurel
+ *          Juni 2018
+ * @license GPL v2+
+ */
+
+#include "BlockNumbering.h"
+#include "chunk_list.h"
+#include "log_levels.h"
+#include "logger.h"
+
+static size_t Number = 0;
+
+
+size_t GetBlockNumber()
```

Comment:
Especially in light of #1843, I wonder where we are planning to go here? At the least, it is a little unusual for functions to start with a capital letter; usually, code that uses CamelCase reserves names with an initial upper-case letter for classes, e.g. `ClassName` and `functionName`.

(And *mixing* CamelCaseWords and words_with_underscores is even more unusual...)